codegen/src/generator/ec2.rs

```diff
@@ -137,7 +137,12 @@ fn generate_response_tag_name<'a>(member_name: &'a str) -> Cow<'a, str> {
 fn generate_method_return_value(operation: &Operation) -> String {
     if operation.output.is_some() {
         let output_type = &operation.output.as_ref().unwrap().shape;
-        let tag_name = generate_response_tag_name(output_type);
+        let standard_tag_name = generate_response_tag_name(output_type).into_owned();
```

Comment:
I'm not enthused about working around the borrow checker like this.  I wonder if a simple `if` would be the way to go.  It'd look like this:

```rust
        let output_type = &operation.output.as_ref().unwrap().shape;
        let mut tag_name = generate_response_tag_name(output_type).into_owned();
        if tag_name == "Snapshot" {
            tag_name = "CreateSnapshotResponse".to_string();
        }
```